.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
don't do this please

Reply:
would you prefer to have a `package-lock.json` in the repository? This is not recommended for libraries!

Reply:
No, just don't put it here. It either belongs in `~/.npmrc` or `.gitignore` but not here.

Reply:
No, I won't put into my `~/.npmrc`, because I normally want a package-lock, but it's not recommended to commit it to library repositories.
`.gitignore` is IMHO the wrong place, too. But if you insist on doing so, I'll do

fyi: https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514

Reply:
I'm very well aware of sindre's specific post as well as the ideas behind this. Initially I opted out of package locks entirely except for specific repos via `~/.npmrc` and then `.npmrc` in the projects that used them. Then I ended up allowing them but including them in `.gitignore` for libs. That way you end up with deterministic, locked versions when working locally but when you push (and CI runs) you don't have locking.

Reply:
If you look down in that thread you'll find that most people choose to ignore lockfiles via gitignore and I've really only seen sindre do `.npmrc` files in module repos.